src/main/java/net/atos/zac/app/informatieobjecten/EnkelvoudigInformatieObjectUpdateService.java

```diff
@@ -60,8 +65,8 @@ public EnkelvoudigInformatieObjectWithLockData updateEnkelvoudigInformatieObject
         EnkelvoudigInformatieObjectLock tempLock = null;
         try {
             final var existingLock = enkelvoudigInformatieObjectLockService.findLock(uuid);
-            if (existingLock.isPresent()) {
-                update.setLock(existingLock.get().getLock());
+            if (existingLock != null) {
```

Comment:
```suggestion
            existingLock?.let {
```

?

Reply:
Refactored the whole block, so now it looks different